benches/coprocessor_executors/mod.rs

```diff
@@ -14,9 +14,13 @@ mod top_n;
 mod util;
 
 fn main() {
+    let mut cpu_set = nix::sched::CpuSet::new();
+    cpu_set.set(0).unwrap();
+    nix::sched::sched_setaffinity(nix::unistd::Pid::from_raw(0), &cpu_set).unwrap();
+
     let mut c = criterion::Criterion::default()
-        .configure_from_args()
-        .sample_size(10);
+        .with_measurement(criterion_papi::PapiMeasurement::new("PAPI_TOT_INS"))
```

Comment:
Prefer to enable it via an env variable so that we can still use time measurement by default

Reply:
OK! Now measurement was set according to `MEASUREMENT` environment variable.